### DISCUSSION TITLE
3D printed parts corrupted

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#108] [Need help]
File for 3D parts are corrupted:

_Unfortunately, it looks like your model file is corrupted. We have detected an abnormally low model volume, which is the result of a corrupt model file and you may need to repair your model for correct prices._

**Comment by sanni** [comment]
Could you specify which file is corrupted? I uploaded them all to Craftcloud3d and did not encounter any error.

**Reply by StanMOI** [reply]
Sorry, 
It's should be my mistake.
Chen I click again in the same link, its works fine !
